models/task/task.go

```diff
@@ -243,19 +244,26 @@ func (u *TaskStruct) ChangeList(listId int64, prevToTaskId *int64, repo *reposit
 			transaction.Rollback()
 			return false
 		}
-		label, err := hub.CheckLabelPresent(token, repo, &listTitle.String)
-		if err != nil {
-			logging.SharedInstance().MethodInfo("task", "ChangeList").Errorf("check label error: %v", err)
-			transaction.Rollback()
-			return false
-		} else if label == nil {
-			// 移動先がない場合はつくろう
-			label, err = hub.CreateGithubLabel(token, repo, &listTitle.String, &listColor.String)
-			if label == nil {
-				logging.SharedInstance().MethodInfo("task", "ChangeList").Errorf("create label error: %v", err)
+
+		var labelName []string
+		if listTitle.String == config.Element("init_list").(map[interface{}]interface{})["none"].(string) {
+			labelName = []string{}
+		} else {
+			label, err := hub.CheckLabelPresent(token, repo, &listTitle.String)
```

Comment:
__[gometalinter]__
* [structcheck] unused struct field cannot pass argument repo (variable of type *../repository.RepositoryStruct) to parameter of type *../../models/repository.RepositoryStruct
* [gotype] undeclared name: hub
* [vetshadow] declaration of err shadows declaration at task.go:229: 
* [aligncheck] cannot pass argument repo (variable of type *../repository.RepositoryStruct) to parameter of type *../../models/repository.RepositoryStruct